src/main/java/com/joelj/jenkins/eztemplates/TemplateProjectListener.java

```diff
@@ -18,7 +19,7 @@ public void onUpdated(Item item) {
         TemplateProperty property = TemplateUtils.getTemplateProperty(item);
```

Comment:
Am going to give Git a workout here... I'm just about to commit a restructure on these listeners that will probably not merge automatically (but is trivial to fix as I'm not changing the signature lines that you are working on)